store/sqlstore/post_store.go

```diff
@@ -1202,3 +1227,57 @@ func (s SqlPostStore) GetOldest() store.StoreChannel {
 		result.Data = &post
 	})
 }
+
+// GetMaxPostSize returns the maximum number of runes that may be stored in a post.
+func (s SqlPostStore) GetMaxPostSize(allowFromCache bool) store.StoreChannel {
```

Comment:
Is `allowFromCache` ever false?

Reply:
Outside of tests, no. I like your suggestions below.